include/boost/asio/detail/impl/reactive_serial_port_service.ipp

```diff
@@ -19,7 +19,11 @@
 #include <boost/asio/detail/config.hpp>
 
 #if defined(BOOST_ASIO_HAS_SERIAL_PORT)
-#if !defined(BOOST_ASIO_WINDOWS) && !defined(__CYGWIN__)
+#if !defined(BOOST_ASIO_WINDOWS) && !defined(__CYGWIN__) 
+
+#ifdef __VXWORKS__
+#include <ioLib.h>
```

Comment:
ioLib. h provides some definitions  that a full POSIX conforming implementation  would provide in stdio.h